troposphere/static/js/components/providers/ProvidersMaster.react.js

```diff
@@ -39,9 +39,15 @@ define(function (require) {
     },
 
     renderProvider: function (provider) {
+      var providers = this.state.providers;
+      var linkName = "provider";
+
+      if (provider === providers.slice(0,1)){ 
+              linkName = "first-provider";
+      }
       return (
         <li key={provider.id}>
-          <Router.Link to="provider" params={{providerId: provider.id}}>
+          <Router.Link to={linkName} params={{providerId: provider.id}}>
```

Comment:
LOL, thought the big title in the description would have been seen. 

Sorry about that, I know it is sloppy. I just wanted to get it to you ASAP and didn't catch extra file until I made the PR. I think it is because of VIM or TMUX (not sure) having that file open from my previous branch.